### DISCUSSION TITLE
naming broken

### DISCUSSION THREAD

**Discussion by alanbret1** [aclap-dev/video-downloadhelper#141] [Help and Q&A]
### Your question

I just want the program to work like is has been now for a good while, until 8.2 
I really hate software updates that don't ask permission before updating--especially when they break the program which was working just fine.
Can I just undo the update somehow?!

**Comment by paulrouget** [comment]
Sorry, we indeed broke naming.

This will be fixed in an upcoming update this weekend.

You can disable autoupdate in Firefox only, in the settings.

**Reply by TLFulton** [reply]
Okay. I came here to ask about issues with naming, but it seems you're on it. For now I have switched all downloading to the CoApp, which is an inconvenience, but one I can tolerate until the coming update. Thanks for all your work on a great app!

**Comment by nooknfi** [comment]
is there a way to downgrade?

**Reply by paulrouget** [reply]
Firefox or Chrome?

**Reply by TLFulton** [reply]
Yes.

**Reply by TLFulton** [reply]
Okay, Firefox.

**Reply by Petervos2018** [reply]
https://addons.mozilla.org/nl/firefox/addon/video-downloadhelper/versions/

**Comment by paulrouget** [comment]
This has been fixed in 8.2.1.

**Reply by TLFulton** [reply]
Sorry, I had been away. I can confirm it is working for me now. Thank you!
